modules/twinklefluff.js

```diff
@@ -356,7 +356,7 @@ Twinkle.fluff.callbacks = {
 				self.params.user = revs[1].getAttribute( 'user' );
 				break;
 			case 'agf':
-				Morebits.status.warn( 'Notice', [ 'Good faith revert was chosen on ', Morebits.htmlNode( 'strong', self.params.user ), '. This is a whitelisted bot, and since bots have no faith, AGF rollback will not proceed.' ] );
```

Comment:
This is hilarious

Reply:
Thinking on it further, hilarity aside this might actually be clearer.  If we want to remove it, maybe something like `This is a whitelisted bot so AGF rollback will not proceed.`

Reply:
Fixed.